### DISCUSSION TITLE
why here is a `position: relative` on `os-viewport`?

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [KingSora/OverlayScrollbars#494] [Q&A] [answered]
Good day! thank you for your excellent work!

I'm writing an announcement page and I want to add a "back to top" button to the bottom left corner of this page. I found that there is a `position: relative` on the `.os-padding.os-viewport` class, which prevents me from positioning the button to the bottom left corner of the entire page. I tried unchecking this and found that it did not have bad effect to the layout of the page, and the button positioning works.

![image](https://user-images.githubusercontent.com/41937333/212476064-b934defa-36c7-4ca8-9a54-d2f036265ead.png)

I'm wondering if this property is necessary? While I can still use `position: fixed` to position the button to the bottom left corner, I would prefer to have the button positioned relative to the container, rather than the entire window.

**Comment by KingSora** [comment, marked as answer]
Good day @Reverier-Xu 

Its needed in certain cases like:
- having padding
- using the absolute padding option
- having a browser or os which isn't supporting scrollbar styling
